tequilapi/endpoints/service.go

```diff
@@ -164,11 +175,40 @@ func (se *ServiceEndpoint) ServiceStart(resp http.ResponseWriter, req *http.Requ
 		return
 	}
 
-	se.serviceState = service.Running
+	optionsParser, ok := se.optionsParser[sr.ServiceType]
+	if !ok {
+		utils.SendErrorMessage(resp, "Invalid service type", http.StatusBadRequest)
+		return
+	}
+
+	options, err := optionsParser(sr.Options)
+	if err != nil {
+		utils.SendErrorMessage(resp, "Invalid options", http.StatusBadRequest)
+		return
+	}
+
+	for _, instance := range se.serviceManager.List() {
```

Comment:
You need some helper function in `servicePool` here

Reply:
Changed it.